JciHitachi/model.py

```diff
@@ -1822,11 +1822,11 @@ def __init__(self, status, default=0):
             "is_numeric": True,
             "legacy_name": "power_kwh",
         },
-        "Ion": {  # currently not supported
-            "controllable": False,
+        "Ion": {
+            "controllable": True,
             "is_numeric": False,
-            "legacy_name": None,
-            "id2str": {},
+            "legacy_name": "ion",
```

Comment:
We don't add "new" legacy names anymore. They exist only for BC reasons that I have been lazy to resolve. Let's just use its regular name `Ion` (and please modify the PR in JciHitachiHA accordingly).

Reply:
You're adding a legacy name `ion` to it, and we don't do that anymore because it is "legacy".

Simply set `None` to `legacy_name` and use its regular name `Ion` instead. You can look at the `Defrost` below as an example.

Reply:
Thanks your response, i modify and push.